src/dynamics/set.jl

```diff
@@ -181,11 +181,22 @@ function set!(
     geometry::Geometry,
     S::Union{Nothing, SpectralTransform}=nothing;
     add::Bool=false,
+    enforce_static_func = false, 
```

Comment:
This kwarg allows to overwrite the explicit transfer to the CPU for functions that are already known at compile time.